business/noticeService.js

```diff
@@ -87,7 +87,15 @@ class NoticeService {
     if (!definition.files) return ''
     const texts = await Promise.all(
       definition.files
-        .filter(file => file.token && file.natures && file.natures.includes('license') && file.path && (file.path.indexOf('/') === -1))
+        .filter(file =>
+          file.token
+          && file.natures
+          && file.natures.includes('license')
+          && file.path
+          && (
+            file.path.indexOf('/') === -1
+            || (definition.coordinates.type === 'npm' && file.path.startsWith('package/'))
```

Comment:
do we want to add some tests for npm or at least update the existing ones for the `package/` case?